dpsim/src/Simulation.cpp

```diff
@@ -177,6 +171,7 @@ void Simulation::sync() {
 	mLog->info("Start synchronization with remotes on {} interfaces", numOfSyncInterfaces);
 
 	for (auto ifm : mInterfaces) {
+		//TODO: Change or document this synchronization behavior
```

Comment:
This is used to synchronize two real-time simulators in the beginning of a co-simulation. During the co-simulation, the two simulators may rely only on their own clocks and no synchronization mechanism to minimize the impact of communication on the real-time performance / variance of the timestep.

Reply:
Knowing that it makes sense to have that mechanism in place, however it was not really documented before how this can be en-/disabled and what exactly it does. Through the comments in the `InterfaceVillas` class, the option to turn it on or off for every attribute is explained now, the exact procedure however (having two dpsim-exports and one import) still remains undocumented.

Reply:
I am collecting open points related to docs in #135